lib/ts/types.ts

```diff
@@ -150,6 +150,8 @@ export type RecipeInterface = {
         apiDomain: string,
```

Comment:
The `DateProvider` needs to be customizable as well. Check out `windowHandler` in the input.